README.md

```diff
@@ -53,16 +60,39 @@ This section describes the terms used for all documentation in this repository.
 
   A particular version of a Linux distribution used to create a rootfs from.
 
-## Usage
+- dracut
+
+  A guest OS build method where the building host is used as the Base OS.
+  For more information refer to the [dracut homepage](https://dracut.wiki.kernel.org/index.php/Main_Page).
+
+## Building
 
 The top-level `Makefile` contains an example of how to use the available components.
 
-By default, components will run on the host system. However, some components
+Two build methods are available, "distro" and "dracut".
```

Comment:
Could you put these terms in backticks? `distro` and `dracut` as they are arguments.